Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2085,6 +2091,36 @@ extension BrowserViewController: TabDelegate {
         updateFindInPageVisibility(visible: true)
         findInPageBar?.text = selection
     }
+    
+    func showRequestRewardsPanel(_ tab: Tab) {
+        let vc = BraveTalkRewardsOptInViewController()
+        
+        let popover = PopoverController(contentController: vc, contentSizeBehavior: .autoLayout)
```

Comment:
Using `autoLayout` will probably break if you end up putting your view inside of a UIScrollView for accessibility reasons. Its probably actually better in the end though to present a regular sheet rather than a popover in accessibility sizes as its easier for users to dismiss and read a sheet rather than a popover

Reply:
Scroll views are not used in both popovers I added
That would require new design and andro/desktop use the popover for it too

But I see your general point is to not introduce any new accessibility debt